tests/unit_tests/pyanaconda_tests/modules/storage/test_module_storage.py

```diff
@@ -904,6 +909,7 @@ def test_mount_kickstart(self, publisher):
         # Mount points configuration
         mount /dev/sda1 /boot
         """
+        self._add_device(StorageDevice("sda1"))
```

Comment:
I wonder why this became needed.

Reply:
`MountPointRequest` now contains the `device_id` so the manual module needs to be able to resolve the device so we need it in the devicetree for the test.